packages/web/DEPRECATIONS-v2.md

```diff
@@ -0,0 +1,120 @@
+# Deprecations List
+
+Following deprecations and feature flags will be removed in version 2 of the _spirit-web_ package.
+
+> Please follow these steps to safely upgrade your design system to Spirit Design System v2 components.
+
+[What are deprecations?][readme-deprecations]
+
+[What are feature flags?][readme-feature-flags]
+
+## Feature Flags
+
+### Dropdown Shadow
+
+The feature flag enabling the dropdown shadow will be removed and the enhanced
+shadow will be enabled by default.
+
+## Modal Uniform Variant
+
+The feature flag enabling the uniform variant of modal will be removed and the
+uniform variant will be enabled by default.
+
+## Tooltip Data Selector Controlled Placement
+
+The feature flag enabling the data selector controlled placement (`data-spirit-placement-controlled`)
+for tooltip will be removed and the data selector controlled placement will be enabled by default.
+
+## Deprecations
+
+## Dropdown Combined Placements
+
+Combined placement classes for dropdown will be removed.
+
+### Migration Guide
+
+Instead of using `.Dropdown--bottom.Dropdown--left`, use `.Dropdown--bottomLeft` and so on.
```

Comment:
Maybe there should be a note saying that if you are using the web-twig or web-react package, this will be done automagically.

Reply:
I don't think so. This file is related only to the web package. It won't appear in the web-twig/react packages readmes and that is sufficient IMHO.

Reply:
Ok.